### ISSUE TITLE
Different positions for bar and handle

### ISSUE THREAD

**Issue by ChristianMayer** [soundar24/roundSlider#23]
It would be great when the position of the bar could be set independently from the slider/handle position like in this mock up:
![slider](https://cloud.githubusercontent.com/assets/10218721/12536464/98a3d610-c2a6-11e5-9e61-c06cd172e0f9.png)

Reason:
Then the bar could show a currently measured value and the handle could display a set point.

**Comment by soundar24** [comment]
I am little bit confused with your requirement.. As you told, the bar will show the current value then the "handle display a set point" means ?
(if i drag the handle then handle should move, then when you need to move the bar ?..)
Can you please update briefly about this..

**Comment by ChristianMayer** [comment]
Ah sorry, this naming is coming from control theory.

Imagine you want to use this to set the room temperature. So you'd use the handle to set it to a position corresponding to the "set point" of the temperature.
As the room doesn't have that temperature yet the heating has to do some work. The currently measured temperature of the room would then be shown by the bar.

At the end the bar position and the handle position must be able to move independently.

**Comment by soundar24** [comment]
Based on your description i created a sample with some workaround.. 
http://jsfiddle.net/soundar24/LpuLe9tr/237/

Just check this demo and confirm me whether this is your required behavior.. And don't worry about the sample level codes, once the behavior confirmed then i will include these in the source.

**Comment by ChristianMayer** [comment]
Generally it's fine.

Only that I can click on the bar and move the bar by mouse is a bit strange (it would be read only in my case). Especially that it's not really deterministic when it's moving the bar and when it's moving the handler. But I guess that's due to the hack :)

**Comment by soundar24** [comment]
Yes, that is due to the hack only.. I slightly changed the sample, now the bar is readonly.. and the bar can be changed through the code only.. And the handle can be move by drag..
http://jsfiddle.net/soundar24/LpuLe9tr/239/

Is this your requirement or you need any improvement in the UI or functionality ?

**Comment by ChristianMayer** [comment]
Great, that's working as required!
